.github/workflows/libcurl.yml

```
@@ -75,9 +71,8 @@ jobs:
           compiler-versions: ${{ matrix.compiler_version }}
           docker-images: ${{ matrix.docker_image }}
         env:
-          CONAN_OPTIONS: "libcurl:shared=True"
           CONAN_SHARED_OPTION_NAME: False
-          CONAN_OPTIONS: ${{ matrix.options }}
+          CONAN_OPTIONS: ${{ matrix.conan_options }}
```

```suggestion
          CONAN_OPTIONS: ${{ matrix.conan_options }}
          CONAN_SHARED_OPTION_NAME: False
```
чтобы статика точно не собиралась

---

так оно же было уже